src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -144,69 +142,12 @@ public AnalyzerResult Analyze()
                     variables,
                     returnType,
                     returnsByRef,
-                    awaitTaskReturn,
                     instanceMemberIsUsed,
                     shouldBeReadOnly,
                     endOfSelectionReachable,
                     operationStatus);
             }
 
-            private (ITypeSymbol typeSymbol, bool awaitTaskReturn) AdjustReturnType(ITypeSymbol returnType)
-            {
-                // if selection contains await which is not under async lambda or anonymous delegate,
-                // change return type to be wrapped in Task
-                var shouldPutAsyncModifier = SelectionResult.CreateAsyncMethod();
-                if (shouldPutAsyncModifier)
-                    return WrapReturnTypeInTask(returnType);
```

Comment:
we no longer do the wrapping step here.  we hold off on that until we need the final type for the method decl to have.